DirectEmployers/MyJobs
# [PD-2951] maps tooltip help text displaying 0

Task was to fix the tooltip data being displayed on the US Map which was initially showing 0 for all of the job views. Now it shows the correct amount of job views.

To Test:

Pull down the repository and filter to get to a US Map. Once you hover over the tooltip should display the correct job views instead of displaying 0 for all of them.
